roles/config-wireguard/config/ymq.yml

```diff
@@ -191,3 +191,10 @@ wg_peers:
   wg_pubkey: mekHV0j+UIFlb166s3BLbult3tsJqgGxzaOv7A0dwWw=
   peer_v4: null
   peer_v6: fe80::2032
+
+- name: dn42-cccihl
+  port: 22451
+  remote: null
+  wg_pubkey: +tsuowf/zrs16pZeModjZGBQzya4tuJImroKh9UfJh0=
+  peer_v4: null
+  peer_v6: fe80::1080
```

Comment:
Please use a peer_v6 that isn't easily confused with my network

e.g. `fe80::<last 4 digits of your ASN>`

Reply:
I have changed the peer_v6 to fe80::2451:1. Please see the latest commit.